### ISSUE TITLE
Include bs-platform version and ppx-flags in report

### ISSUE THREAD

**Issue by woeps** [ryyppy/re-compiler-benchmark-tool#2]
If the report were to include the bs-platform version and ppx-flags the data could be put in relation more accurately.

Also: It could be of interest to additionally build everything (including dependencies) and include the list of bs-dependencies and the compilation time in the report.

**Comment by ryyppy** [comment]
Yeah that is a good idea. I also wanted to integrate the `bstracing` output somehow so we also have access to the flamegraph (see [docs](https://rescript-lang.org/docs/manual/latest/build-performance))
